src/Components/Categories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Navbar from './Navbar';
+
+function Categories() {
+  return (
+    <div>
+      <Navbar />
+      <div className="container">
+        <h2>Book Catagoris!</h2>
```

Comment:
Categories should display only a button with text "Check status", Kindly fix this issue 👍